cime_config/testdefs/testlist_clm.xml

```diff
@@ -1996,7 +1996,7 @@
       <option name="wallclock">00:20:00</option>
     </options>
   </test>
-  <test name="SMS_D_Ld1" grid="ne30pg3_t061" compset="I1850Clm50BgcSpinup" testmods="clm/cplhist--clm/matrixcnOn">
+  <test name="SMS_D_Ld1" grid="ne30pg3_t232" compset="I1850Clm50BgcSpinup" testmods="clm/cplhist--clm/matrixcnOn">
```

Comment:
In the same file, I see t061 in this line:
`  <test name="SMS_Ln9" grid="C96_t061" compset="I2000Clm60Sp" testmods="clm/clm60cam7LndTuningMode--clm/nofireemis">`
BUT I do not mention of this grid in the list of failures.

Reply:
Let's fix this one one too though. It did pass in ctsm_sci as is. But, t061 is being phased out, so we might as well change now.

Reply:
@ekluzek 
From the error that I got, it appears as though this one requires a submodule update in `ccs_config/modelgrid_aliases_nuopc.xml`. Do you still want me to pursue it or open an issue?